### PR TITLE
ECS-3078 review certificates.md, the description of the certificate c…

### PR DESCRIPTION
**Pull request by NeilBurgess42**
…hain.

## Content

Apply style guide to the document, suggest some changes to make it more concise.

## Pre-submit checklist

- Branch
  - [x] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [x] Add dev blog post (if relevant)

## Issue(s)

Relates to #YYY or Closes #YYY

### CODE REVIEW

**Review by olgahryniuk — APPROVED** [review]

**Comment by olgahryniuk** [comment]
@jpraynaud please review

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 1]
```diff
@@ -3,77 +3,76 @@ sidebar_position: 2
...
+  
+The **certificate chain** is a Mithril component that certifies the **stake distribution** used to create the multi-signature. Its primary purpose is to prevent adversaries from executing an **eclipse attack** on the blockchain.  
 
-The **stake distribution** that is used to create the multi signature can't be trusted "as is" and must also be signed. Indeed, any one could relatively easily create a **Fake Stake Distribution** and use it to produce a valid multi signature that would be embedded in a perfectly **Valid but Non Genuine Certificate**. This certificate could be served by a dishonest **Mithril Aggregator** node and thus could lead to restoring a non genuine snapshot by a honest **Mithril Client**.
+Without the certificate, the stake distribution can't be trusted. A malicious actor could relatively easily create a fake stake distribution and use it to produce a valid multi-signature, which would be embedded in a valid but non-genuine certificate. This certificate could be served by a dishonest Mithril aggregator node, leading an honest Mithril client to restore a non-genuine snapshot. That is an eclipse attack.
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:16]:
```suggestion
Without the certificate, the stake distribution can't be trusted. A malicious actor could relatively easily create a fake stake distribution and use it to produce a valid multi-signature, which would be embedded in a valid but non-genuine certificate. This certificate could be served by a dishonest Mithril aggregator node, leading an honest Mithril client to restore a non-genuine snapshot.  
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 2]
```diff
@@ -3,77 +3,76 @@ sidebar_position: 2
...
 
-Also, the certificate at the beginning of the **Certificate Chain** has a special role. This is the **Genesis Certificate**. The only way to validate the stake distributon embedded in the **Genesis Certificate** is to sign it with a private key that is associated to a broadly available public key: the **Genesis Keys**. These are the keys that sign the hard forks used during the era transitions of the Cardano Blockchain.
+The first certificate in the certificate chain is known as the **genesis certificate**. Validating the stake distribution embedded in the genesis certificate is only possible by signing it with a private key linked to a widely accessible public key called the **genesis key**. These keys are responsible for signing the hard forks used during the era transitions of the Cardano blockchain. The use of these specific keys ensures the integrity and security of the initial stake distribution and subsequent transitions within the blockchain network.
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:29]:
```suggestion
The first certificate in the certificate chain is known as the **genesis certificate**. Validating the stake distribution embedded in the genesis certificate is only possible by signing it with a private key linked to a widely accessible public key called the **genesis key**. The use of these specific keys ensures the integrity and security of the initial stake distribution and subsequent transitions within the blockchain network.
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 3]
```diff
@@ -3,77 +3,76 @@ sidebar_position: 2
...
-The informations embedded in the `METADATA(p,n)` field are:
+Information embedded in the `METADATA(p,n)` field:
 
-* The version of the Mithril Protocol
-* The parameter of the Mithril Protocol (`k`, `m`, and `phi_f`)
-* The date and time at which the multi signature creation was initiated
-* The date and time at which the Certificate was sealed
-* The list of the signers that actively contributed to the multi signature
+* The version of the Mithril protocol
+* The parameter of the Mithril protocol (`k`, `m`, and `phi_f`)
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:55]:
```suggestion
* The parameters of the Mithril protocol (`k`, `m`, and `phi_f`)
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 4]
```diff
@@ -3,77 +3,76 @@ sidebar_position: 2
...
+The message `MSG(p,n)` is a map of multiple values associated with their respective keys. It provides a way to add more information to the certificates without breaking the chain itself. Added items can be any message that the signers can compute deterministically thanks to the Cardano consensus – an immutable files snapshot, the UTXO set, stake distribution, etc.
 
 :::note
 
-The **trigger** represents the instant at which a certificate should be created. It is combined, with at least the associated **epoch** to create the [**Beacon**](../../glossary.md#beacon) of the certificate. In the current implementation, this trigger is a new [**Immutable File Number**](../../glossary.md#immutable-file-number). The epoch creation and the trigger creation processes may be uncorrelated.
+The **trigger** represents the instant at which a certificate should be created. It is combined with at least the associated **epoch** to create a [**beacon**](../../glossary.md#beacon) of the certificate. In the current implementation, this trigger is a new [**immutable file number**](../../glossary.md#immutable-file-number). The epoch creation and the trigger creation processes may be uncorrelated.
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:64]:
```suggestion
The **trigger** represents the instant at which a certificate should be created. It is combined with at least the associated **epoch** to create a [**beacon**](../../glossary.md#beacon) of the certificate. In the current implementation of the Cardano node database snapshot, this trigger is a new [**immutable file number**](../../glossary.md#immutable-file-number). 
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 5]
```diff
@@ -82,49 +81,49 @@ CHAIN_VERIFY[C(p,n(p))] = CERT_VERIFY[C(p,n(p)] ^ CERT_VERIFY[FC(n(p))] ^ CERT_V
...
+A **non-genesis certificate** is valid if and only if the `AVK` used to verify the multi-signature is also part of the signed message used to create a valid multi-signature in a previously sealed certificate.
 
-The **Genesis Certificate** is valid **if and only if** its **Genesis Signature** is verified with the advertised **Public Genesis Key**.
+The genesis certificate is valid if and only if its genesis signature is verified with the advertised public genesis key.
 
 An implementation of the algorithm would work as follows for a certificate:
 
 * **Step 1**: Use this certificate as the `current_certificate`
-* **Step 2**: Verify (or fail) that the `current_hash` of the `current_certificate` is valid (by computing it and comparing with the `hash` field of the certificate)
+* **Step 2**: Verify (or fail) that the `current_hash` of the `current_certificate` is valid by computing it and comparing it with the `hash`` field of the certificate
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:96]:
```suggestion
* **Step 2**: Verify (or fail) that the `current_hash` of the `current_certificate` is valid by computing it and comparing it with the `hash` field of the certificate
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 6]
```diff
@@ -82,49 +81,49 @@ CHAIN_VERIFY[C(p,n(p))] = CERT_VERIFY[C(p,n(p)] ^ CERT_VERIFY[FC(n(p))] ^ CERT_V
...
 * **Step 3**: Get the `previous_hash` of the `previous_certificate` by reading its value in the `current_certificate`
 * **Step 4**: Verify (or fail) that the `multi_signature` of the `current_certificate` is valid
 * **Step 5**: Retrieve the `previous_certificate` that has the hash `previous_hash`:
   * **Step 5.1**: If it is not a `genesis_certificate`:
-    * **Step 5.1.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid (by computing it and comparing with the `hash` field of the certificate)
-    * **Step 5.1.2**: Verify the `current_avk`
-      * **Step 5.1.2.1**: If the `current_certificate` is the `first_certificate` of the epoch, verify (or fail) that the `current_avk` of the `current_certificate` is part of the message signed by the multi signature of the `previous_certificate`
+    * **Step 5.1.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid by computing it and comparing it with the `hash`` field of the certificate:
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:101]:
```suggestion
    * **Step 5.1.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid by computing it and comparing it with the `hash` field of the certificate:
```

File: docs/root/mithril/mithril-protocol/certificates.md [hunk 7]
```diff
@@ -82,49 +81,49 @@ CHAIN_VERIFY[C(p,n(p))] = CERT_VERIFY[C(p,n(p)] ^ CERT_VERIFY[FC(n(p))] ^ CERT_V
...
+    * **Step 5.1.2**: Verify the `current_avk`:
+      * **Step 5.1.2.1**: If the `current_certificate` is the `first_certificate` of the epoch, verify (or fail) that the `current_avk` of the `current_certificate` is part of the message signed by the multi-signature of the `previous_certificate`
       * **Step 5.1.2.2**: Else verify (or fail) that the `current_avk` of the `current_certificate` is the same as the `current_avk` of the `previous_certificate`
     * **Step 5.1.3**: Verify (or fail) that the `multi_signature` of the `previous_certificate` is valid
     * **Step 5.1.4**: Use the `previous_certificate` as `current_certificate` and start again at **Step 2**
   * **Step 5.2**: If it is a `genesis_certificate`:
-    * **Step 5.2.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid (by computing it and comparing with the `hash` field of the certificate)
+    * **Step 5.2.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid by computing it and comparing it with the `hash`` field of the certificate
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/certificates.md:108]:
```suggestion
    * **Step 5.2.1**: Verify (or fail) that the `previous_hash` of the `previous_certificate` is valid by computing it and comparing it with the `hash` field of the certificate
```

**Review by jpraynaud — CHANGES_REQUESTED** [review]
Few minor changes before it's ready to be merged :slightly_smiling_face:

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Pull request merged**